presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -6,8 +6,8 @@ Presto supports statistics based optimizations for queries. For a query to take
 advantage of these optimizations, Presto must have statistical information for
 the tables in that query.
 
-Table statistics are provided to the query planner by connectors.  Currently, the
-only connector that supports statistics is the :doc:`/connector/hive`.
+Table statistics are provided to the query planner by connectors. Support for
```

Comment:
`Support for statistics is dependent upon the connector.` - this is confusing to me. What we want to convey is that connectors can choose to provide/not provide statistics, Presto itself does not fetch or generate any statistics for tables at runtime